build.gradle

```diff
@@ -10,8 +10,8 @@ version = '2.3.0'
 
 description = """Imgix Java Client"""
 
-sourceCompatibility = 11
-targetCompatibility = 11
```

Comment:
This compat is needed when building locally but breaks the travis tests. Unfortunate, but until travis versions change this has to be left alone.